Assets/MixedRealityToolkit/_Core/Definitions/SpatialAwareness/SpatialAwarenessSurfaceTypes.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.SpatialAwareness
+{
+    /// <summary>
+    /// Flags that indicate the semantic(s) for a surface in the environment.
+    /// </summary>
+    [Flags]
```

Comment:
not sure if flags are appropriate here.

Probably shouldn't be able to define a wall and ceiling, and we might want to split this up into two enums for describing the surfaces (hangable, angled, etc)

Reply:
valid point

Reply:
In plane finding, these were used as flags to allow us to specify which types of planes to identify / keep... the new system may want that, but i am willing to return the location and dimensions of all surfaces and let the application construct whichever ones they wish.